src/bundle/Resources/views/content/modal_location_bulk_operation_failed.html.twig

```diff
@@ -0,0 +1,39 @@
+{% trans_default_domain 'locationview' %}
+
+<div
+    class="modal fade ez-bulk-operation-failed-modal"
```

Comment:
Why do you duplicate `id` as a class name?

Reply:
I think that if we want to be compliant with BEM and use classes for elements (like `ez-bulk-operation-failed-modal__table-body`) we need class for whole block. I also use this class to style modal.

Reply:
I checked how it works in other places in our codebase and it seems that we have convention to give `id`s to modals and later refer to them by these ids. It also matches examples from https://getbootstrap.com/docs/4.0/components/modal/.

But I don't use id in my piece of code, instead I have class selector:

```js
const SELECTOR_BULK_OPERATION_FAILED_MODAL = '.ez-bulk-operation-failed-modal';
```

I think that the right way would be to change above selector to:

```js
const SELECTOR_BULK_OPERATION_FAILED_MODAL = '#bulk-operation-failed-modal'
```

and leave `class` for styling puroposes. 

Are you OK with this? 🙂